util/kubeconfig/kubeconfig_test.go

```diff
@@ -85,19 +84,17 @@ func setupScheme() *runtime.Scheme {
 }
 
 func TestGetKubeConfigSecret(t *testing.T) {
+	g := NewWithT(t)
+
 	clusterKey := client.ObjectKey{
 		Name:      "test1",
 		Namespace: "test",
 	}
 	client := fake.NewFakeClientWithScheme(setupScheme(), validSecret)
-	found, err := FromSecret(context.Background(), client, clusterKey)
-	if err != nil {
-		t.Fatalf("Expected no error, got %v", err)
-	}
 
-	if !reflect.DeepEqual(validSecret.Data[secret.KubeconfigDataName], found) {
-		t.Fatalf("Expected found secret to be equal to input")
-	}
+	found, err := FromSecret(context.Background(), client, clusterKey)
+	g.Expect(err).NotTo(HaveOccurred())
+	g.Expect(validSecret.Data[secret.KubeconfigDataName]).To(Equal(found))
```

Comment:
Nit: Normally you `Expect` the thing you got from the function you are testing to `Equal` a known value, helps the readability when there's an error
```suggestion
	g.Expect(found).To(Equal(validSecret.Data[secret.KubeconfigDataName]))
```